apps/web/components/Templates/Intro.tsx

```diff
@@ -0,0 +1,12 @@
+export default function Intro() {
+  return (
+    <>
+      <h2 className="text-left text-4xl font-medium xl:text-5xl">Templates</h2>
```

Comment:
H2 styles would probably be repeating in the app quite often, consider creating a Typography component, just like it is in MUI